### ISSUE TITLE
Getters/setters shouldn't trigger no-dupe-keys

### ISSUE THREAD

**Issue by gcurtis** [eslint/eslint#634]
The following code triggers the no-dupe-keys rule in v0.4.0:

``` javascript
var foo = {
    get prop() {
        return this.p;
    },
    set prop(val) {
        this.p = val;
    }
};
```

``` bash
$ eslint foo.js

foo.js
  1:10  error  Duplicate key 'prop'            no-dupe-keys
  2:12  error  Missing "use strict" statement  strict
  5:15  error  Missing "use strict" statement  strict

✖ 3 problems
```

It seems that the rule doesn't consider getters/setters declared in object initializers.
